resources/webroot/stats.mustache

```diff
@@ -852,11 +853,10 @@
                 }
             });
         }
-
     });
 </script>
 
-<h2 class="page-header text-center">{{title}}'s {{title_stats}}</h2>
+<h2 class="page-header text-center">{{page}}</h2>
```

Comment:
Is page a translation?

Reply:
Yup!

![image](https://user-images.githubusercontent.com/1654959/168440570-fb989553-27a2-4222-88e6-6c5aad524b31.png)